### ISSUE TITLE
CloudFormation now supports HTTP -> HTTPS redirection

### ISSUE THREAD

**Issue by dougireton** [jheller/alb-rule#2]
Here's the snippet:

```yams
  HTTPlistener:
    Type: AWS::ElasticLoadBalancingV2::Listener
    Properties:
      DefaultActions:
        - Type: redirect
          RedirectConfig:
            Protocol: HTTPS
            Host: "#{host}"
            Query: "#{query}"
            Path: /#{path}
            Port: "443"
            StatusCode: HTTP_301
      LoadBalancerArn: !Ref MyLoadBalancer
      Port: 80
      Protocol: HTTP
```

**Comment by jheller** [comment]
Great new! Have you tried it with other RedirectConfig settings? E.g. redirecting /images to another host.

I am still a bit confused looking at the online documentation. Although RedirectConfig is now listed in the documentation as valid when Type is **redirect**, the entry for Type says **forward** is the only valid value.

**Comment by dougireton** [comment]
The CloudFormation docs aren't caught up with CloudFormation functionality. I submitted this PR to add `redirect` as a valid type: https://github.com/awsdocs/aws-cloudformation-user-guide/pull/199

I haven't tried redirecting images yet.

**Comment by jheller** [comment]
I'm guessing that other rediects will also work now. That makes this Lambda function no longer necessary.

Thanks for the tip-off. :-)
